resources/lang/id/beatmaps.php

```diff
@@ -0,0 +1,282 @@
+            'disqualify' => 'Didiskualifikasi :time_ago oleh :user akibat ditemukannya masalah baru berupa :discussion (:message).',
+        ],
+
+        'reset_confirm' => [
+            'nomination_reset' => 'Apakah kamu yakin? Memposting masalah baru akan mengulang proses nominasi.',
+        ],
+    ],
+
+    'listing' => [
+        'search' => [
+            'prompt' => 'ketik kata kunci...',
+            'options' => 'More Search Options',
+            'not-found' => 'tidak ada hasil',
+            'not-found-quote' => '... tidak, tidak ditemukan apa pun.',
+            'filters' => [
+                'general' => 'Umum',
+                'mode' => 'Mode',
+                'status' => 'Status Map',
+                'genre' => 'Aliran',
+                'language' => 'Bahasa',
+                'extra' => 'tambahan',
+                'rank' => 'Peringkat yang Diraih',
+                'played' => 'Telah Dimainkan',
+            ],
+        ],
+        'mode' => 'Mode',
+        'status' => 'Status Map',
+        'mapped-by' => 'dibuat oleh :mapper',
+        'source' => 'dari :source',
+        'load-more' => 'Muat lebih banyak...',
+    ],
+    'general' => [
+        'recommended' => 'Tingkat kesulitan yang disarankan',
+        'converts' => 'Sertakan beatmap konversi',
+    ],
+    'mode' => [
+        'any' => 'Apa saja',
+        'osu' => 'osu!',
+        'taiko' => 'osu!taiko',
+        'fruits' => 'osu!catch',
+        'mania' => 'osu!mania',
+    ],
+    'status' => [
+        'any' => 'Apa saja',
+        'ranked-approved' => 'Ranked & Approved',
+        'approved' => 'Approved',
+        'qualified' => 'Qualified',
+        'loved' => 'Loved',
+        'faves' => 'Favorit',
+        'pending' => 'Pending',
+        'graveyard' => 'Graveyard',
+        'my-maps' => 'Map Saya',
+    ],
+    'genre' => [
+        'any' => 'Apa saja',
+        'unspecified' => 'Tidak Terdefinisi',
+        'video-game' => 'Video Game',
+        'anime' => 'Anime',
+        'rock' => 'Rock',
+        'pop' => 'Pop',
+        'other' => 'Lainnya',
+        'novelty' => 'Novelty',
+        'hip-hop' => 'Hip Hop',
+        'electronic' => 'Electronic',
+    ],
+    'mods' => [
+        '4K' => '4K',
+        '5K' => '5K',
+        '6K' => '6K',
+        '7K' => '7K',
+        '8K' => '8K',
+        '9K' => '9K',
+        'AP' => 'Auto Pilot',
+        'DT' => 'Double Time',
+        'EZ' => 'Easy Mode',
+        'FI' => 'Fade In',
+        'FL' => 'Flashlight',
+        'HD' => 'Hidden',
+        'HR' => 'Hard Rock',
+        'HT' => 'Half Time',
+        'NC' => 'Nightcore',
+        'NF' => 'No Fail',
+        'NM' => 'No mods',
+        'PF' => 'Perfect',
+        'Relax' => 'Relax',
+        'SD' => 'Sudden Death',
+        'SO' => 'Spun Out',
+        'TD' => 'Touch Device',
+    ],
+    'language' => [
+        'any' => 'Apa saja',
+        'english' => 'English',
```

Comment:
Inggris (juga untuk bahasa lain).

Reply:
Should i add prefix "Bahasa" for each?

Reply:
The row header already says Language (or Bahasa if it's ever localized) so it should be fine without.